zrml/prediction-markets/src/lib.rs

```diff
@@ -1322,6 +1381,7 @@ mod pallet {
         /// Shares of outcome assets and native currency
         type AssetManager: ZeitgeistAssetManager<
             Self::AccountId,
+            Balance = <CurrencyOf<Self> as Currency<Self::AccountId>>::Balance,
```

Comment:
`BalanceOf<T>` is (surprisingly? confusingly?) defined using the `AssetManager` and not the `Currency`, so this will probably (hopefully?) not compile. The point is that we want to communicate that the `Balance` type of the `AssetManager` must be the same as the `Balance` already defined by the `Currency` trait.